server/memcached/src/test/java/org/infinispan/server/memcached/MemcachedClusteredStatsTest.java

```diff
@@ -45,8 +46,16 @@ public EmbeddedCacheManager createCacheManager(int index) {
    public void testSingleConnectionPerServer() throws MalformedObjectNameException, AttributeNotFoundException,
          MBeanException, ReflectionException, InstanceNotFoundException {
       MBeanServer mBeanServer = mbeanServerLookup.getMBeanServer(null);
-      ConnectionStatsTest.testGlobalConnections(jmxDomain + "-0", "Memcached-" + jmxDomain, 2,
-            mBeanServer);
+      ObjectName on = new ObjectName(String.format("%s:type=Server,name=%s,component=Transport", jmxDomain + "-0", "Memcached-" + jmxDomain));
+      // Now verify that via JMX as well, these stats are also as expected
+      eventuallyEquals(2, () -> {
```

Comment:
Any idea why the delay is needed here but not on master? 

If it's not actually related to the the dist executor removal, I'd rather do it in a separate PR, because I think the test needs more cleanup:
* The test method is named "testSingleConnectionPerServer", but we expect 2 connections?
* The "JMX as well" comment is misleading, we only ever check JMX. Can we get the stats without JMX?
  * It was likely copied from `ConnectionStatsTest.testSingleLocalConnection()`, but it doesn't make sense there either
* All the methods in `ConnectionStatsTest` should be named `assertX`, not `testX`

Reply:
odd..it passed every time for me on IDE

My guess is the connection is established asynchronously and the test must be running a little quicker on CI.

The other option is I could actually fire an operation with the connection, which would guarantee it is connected. However these are done async as well, so I would still have to add in a wait somewhere. I figured this was just a bit easier.

Reply:
@wburns do we know how spymemcached selects the server to execute an operation? If not, we can't guarantee each server is connected before asserting. Personally I'd just delete the test, because we don't actually control the number of connections, but I think I've delayed this PR enough :)